src/net.cpp

```diff
@@ -1640,6 +1627,10 @@ void CConnman::ThreadDNSAddressSeed()
                     found++;
                 }
                 addrman.Add(vAdd, resolveSource);
+            } else {
+                // We now just ignore DNS Seeds which do not support service
```

Comment:
This branch triggers based on SetInternal returning false, which means host="" - it has nothing to do with service filtering.

Reply:
I think you reviewed with too little context? Generally -U20 or more is required to get sufficient context in reviews in my experience :p.

Reply:
Oops, indeed.